pom.xml

```diff
@@ -82,7 +81,7 @@
     <scala.maven.test.skip>false</scala.maven.test.skip>
     <pig.version>0.16.0</pig.version>
     <pig.classifier>h2</pig.classifier>
-    <thrift.version>0.7.0</thrift.version>
+    <thrift.version>0.9.3</thrift.version>
```

Comment:
Could you please also exclude the Thrift version in the exclude lines? Replacing them with ${thrift.version} would be the best.